site/gdocs/components/LatestDataInsights.tsx

```diff
@@ -251,10 +255,12 @@ function useDotButton(emblaApi: EmblaCarouselType | undefined): {
 
 function DotButton({
     children,
+    index,
     ...restProps
-}: ComponentPropsWithRef<"button">) {
+}: ComponentPropsWithRef<"button"> & { index: number }) {
```

Comment:
Actually, I was meaning to change this to a div instead of a button and remove the interactivity. The buttons are too small for touch navigation — it was flagged as such by some other web test tool I used. Should we do that instead?